library/src/main/java/org/apache/apex/malhar/lib/state/spillable/SpillableMapImpl.java

```diff
@@ -236,13 +237,27 @@ public void beginWindow(long windowId)
   @Override
   public void endWindow()
   {
+    boolean isTimeUnifiedStore = (store instanceof ManagedTimeUnifiedStateImpl);
```

Comment:
As it added one member variable, So it increase the complexity. I don't think it simplify the logic, as the variable expand it's scoop.